app/helpers/organisation_helper.rb

```diff
@@ -2,4 +2,31 @@ module OrganisationHelper
   def child_organisations_count(organisation)
     organisation["works_with"].values.flatten.count
   end
+
+  def child_organisation_count(organisation)
```

Comment:
We've got this method, and a `child_organisation_count` method in the `organisation` model.  Do we need both of them?

Reply:
The method in organisation model is removed now, instead organisation_helper handles this 👍